### DISCUSSION TITLE
Account Restricted

### DISCUSSION THREAD

**Discussion by johndoe-cmyk** [cloudbet/docs#84] [Q&A] [answered]
Hello, I came here because the customer support did send me this:
Conversation with Cloudbet
Started on June 28, 2024 at 10:30 AM Amsterdam time CEST (GMT+0200)

---

10:30 AM | disguisedCare: [Image "image.png?expires=1719570600&signature=f13fa7bcba3f6eaa2df54d36593d0f060d99ea517e993740ef4b515c157bccef"]

10:31 AM | disguisedCare: Hey, I did the second stage of KYC with my driving license and I still get this response from the API when I try to automatically bet on a tennis match: 
 RESTRICTED - your current account settings don't allow you to bet on this event. Restrictions will be lifted automatically as your account attains tenure and trust. Please contact customer support if you believe you qualify and we will review your account. 
 The documentation said I have to contact customer support as I did all the KYC processes, so I should be unlocked for automatically betting. Could you arrange this please or do you need anything else (passport and so on? ) - I'm willing to provide every document needed!

11:04 AM | Eva from Cloudbet: Hello! Welcome to Cloudbet support disguisedCare, I'm Eva and I will be helping you out with your query today.

11:04 AM | Eva from Cloudbet: Apologise for the delay disguisedCare.

11:05 AM | Eva from Cloudbet: Was KYC requested for your account disguisedCare?

11:05 AM | Eva from Cloudbet: I believe KYC was not necessary at this point.

11:06 AM | Eva from Cloudbet: Regarding API, please visit the API discussions forum at: https://github.com/Cloudbet/docs/discussions (https://github.com/Cloudbet/docs/discussions) 
 
 They have a dedicated team of experts there who will be better equipped to assist you with your query disguisedCare. 
 
﻿​﻿

Can you help me to resolve the restriction?

Kind regards,
Jeremy

**Comment by kgravenreuth** [comment]
Hi @johndoe-cmyk 

After reviewing your account, please note that Access from restricted countries as per [Cloudbet Terms & Conditions](https://www.cloudbet.com/en/help/terms?section=71-your-account) is blocked.

Note that such country restrictions are imposed both at account registration as well as bet placement times and depends on the location of your API client. This is also mentioned in our [Trading API Docs](https://www.cloudbet.com/api/?urls.primaryName=Trading)

Let me know if this clarifies your question, cheers.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by johndoe-cmyk** [reply]
Hi @kgravenreuth ,

Thanks for your answer. My home country is not on the block list, but the server my bot runs on is in Germany, so that is the case, am I correct? :)
If I host the app on my server at home, everything should work correctly, am I correct?


Kind regards,
Jeremy

**Reply by kgravenreuth** [reply, marked as answer]
Hi Jeremy,
If your betting requests are from an unrestricted country, then you should not see such bet restricted errors from a Cloudbet API point of view.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by johndoe-cmyk** [reply]
Thanks a lot!!!
